cmd/better-retester/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os"
+	"testing"
+)
+
+func stringSlicesEqual(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i, v := range a {
+		if v != b[i] {
+			return false
+		}
+	}
+	return true
+}
+func TestGatherOptions(t *testing.T) {
+	expected := []string{"org/repo", "other-org/other-repo"}
+	os.Args = []string{"cmd", "--enable-on-repo=org/repo", "--cache-record-age=100h", "--enable-on-repo=other-org/other-repo"}
+
+	actual := gatherOptions()
+
+	if !stringSlicesEqual(actual.enableOnRepos.Strings(), expected) {
```

Comment:
You should just use `cmp.Diff` like all other unit  tests do: https://pkg.go.dev/github.com/google/go-cmp/cmp#Diff